routes/api/users.js

```diff
@@ -79,7 +81,17 @@ router.post("/users", function(req, res, next) {
 
     user.save()
         .then(function() {
-            return res.json({ user: user.toAuthJSON() });
+            console.log(TokenService);
+            TokenService.createToken(req.body.user, '3d', function(token){
```

Comment:
first off! transform this to either return just the token and not make use of a callback cos then you're now forced to return a response in the context of the `TokenService.createToken`. What if you still had to do something else with the user data.

Also, define the token expiry inside where you're using it, cos it's redundant if you have to pass in a default expiry value to the function, and that just makes the function impure.

Finally, I'd like to hope that since you're creating the token with the whole `req.body.user` that the user's password ain't in there somehow